x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/risk_badge.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import { RiskSeverity } from '../../../common/search_strategy';
+import { RISK_SEVERITY_COLOUR } from '../../entity_analytics/common/utils';
+import { getRiskLevel } from '../../../common/entity_analytics/risk_engine/risk_levels';
+
+export interface RiskBadgeProps {
+  risk: number;
+  'data-test-subj'?: string;
+}
+
+const tooltips = {
+  [RiskSeverity.Unknown]: i18n.translate(
+    'xpack.securitySolution.assetInventory.allAssets.risks.unknown',
+    {
+      defaultMessage: RiskSeverity.Unknown,
+    }
```

Comment:
minor nitty comment, but those can be inlined automatically by eslint, just need to remove the blank space before the "defaultMessage" part until the start of the brackets
```suggestion
    { defaultMessage: RiskSeverity.Unknown }
```

(repeat for other translations)

Reply:
@JordanSh That was prettier, but I changed it manually and it wasn't reverted :)